doc/managing-the-glusterd2-service.md

```diff
@@ -0,0 +1,54 @@
+# Managing the glusterd Service
+
+After installing GlusterFS and Glusterd2, you must start `glusterd2`
+service. The glusterd2 service serves as the Gluster elastic volume
+manager, overseeing glusterfs processes, and co-ordinating dynamic
+volume operations, such as adding and removing volumes across multiple
+storage servers non-disruptively.
+
+This section describes how to start the glusterd2 service in the
+following ways:
+
+**Note**: You must start glusterd2 on all GlusterFS servers.
+
+## Gluster user group
+
+On each server nodes, create "gluster" user group if not exists
+already using the following command.
+
+    # groupadd gluster
+
+Users from this group can run `glustercli` commands in GlusterFS
```

Comment:
Nit: It seems more idomatic to me to say "on GlusterFS servers" rather than "in"